src/sql/azdata.d.ts

```diff
@@ -194,6 +194,7 @@ declare module 'azdata' {
 		 * @param connectionProfile The {@link IConnectionProfile} containing the information for the connection
 		 * @param saveConnection Whether to save the connection in the saved connections list of the Servers view. Default is true
 		 * @param showDashboard Whether to show the dashboard for the connection upon success. Default is true
+		 * @param ownerUri The URI string of the document that owns the resulting connection. Default is undefined
```

Comment:
Forget to add the overload?

Reply:
Had one before, but removed it and forgot to update the doc comment.